src/features/defi/providers/fox-eth-lp/components/FoxEthLpManager/Overview/FoxEthLpOverview.tsx

```diff
@@ -23,21 +31,46 @@ export const FoxEthLpOverview: React.FC<FoxEthLpOverviewProps> = ({
   accountId,
   onAccountIdChange: handleAccountIdChange,
 }) => {
-  const {
-    foxEthLpOpportunity: opportunity,
-    lpFoxBalance: foxBalance,
-    lpEthBalance: ethBalance,
-    lpLoading: loading,
-  } = useFoxEth()
+  const { query } = useBrowserRouter<DefiQueryParams, DefiParams>()
+  const { highestBalanceAccountAddress } = query
+
+  const accountAddress = useMemo(
+    () => (accountId ? fromAccountId(accountId ?? '').account : ''),
+    [accountId],
+  )
+
+  const opportunity = useAppSelector(state =>
+    selectFoxEthLpOpportunityByAccountAddress(state, {
+      accountAddress: accountAddress ?? '',
+    }),
+  )
+
+  // Making sure we don't display empty state if account 0 has no farming data for the current opportunity but another account has
+  useEffect(() => {
+    if (highestBalanceAccountAddress && accountAddress !== highestBalanceAccountAddress) {
+      const highestBalanceAccountId = toAccountId({
+        account: highestBalanceAccountAddress,
+        chainId: ethChainId,
+      })
+      handleAccountIdChange(highestBalanceAccountId)
+    }
+    // This should run only once, else we won't be able to select another account than the defaulted highest balance one
```

Comment:
Typo

Reply:
`defaulted` is actually the intended verbiage, we *default* the AccountId as an action, but we don't actually have a *default* account (e.g a default account we would pass from props)